tests/phpunit/Checker/Default_Check_Repository_Tests.php

```diff
@@ -115,4 +117,80 @@ public function test_get_checks_throws_exception_for_invalid_check_slug() {
 
 		$this->repository->get_checks( Check_Repository::TYPE_ALL, array( 'invalid_check' ) );
 	}
+
+	public function test_get_checks_returns_no_experimental_checks_by_default() {
+		$static_check       = new Static_Check();
+		$runtime_check      = new Runtime_Check();
+		$experimental_check = new Experimental_Static_Check();
+
+		$this->repository->register_check( 'static_check', $static_check );
+		$this->repository->register_check( 'runtime_check', $runtime_check );
+		$this->repository->register_check( 'experimental_static_check', $experimental_check );
```

Comment:
Nit-pick, but since you register both `Experimental_Static_Check` and `Experimental_Runtime_Check` in all other tests, I'd suggest you do that here too. Shouldn't change the expected result, but the fact that it's currently missing here makes me wonder whether it's intentional for some reason.

Reply:
Thanks @felixarntz I've included the check in the test